### ISSUE TITLE
Save wakatime-cli to home folder issue

### ISSUE THREAD

**Issue by QuentiumYT** [wakatime/vscode-wakatime#189]
Hello,
I'm having an issue with the new release v6.0.3 (02d698257d483528e57d2d3247c058abaeca0f14) with the new feature that cause extensions to fail loading:

```
[Error: ENOENT: no such file or directory, open
'C:\Users\MyUserName\.wakatime\wakatime-cli.zip'] {
    errno: -4058,
    code: 'ENOENT',
    syscall: 'open',
    path: 'C:\\Users\\MyUserName\\.wakatime\\wakatime-cli.zip'
}
```

As the error stated, I can tell that I don't have any wakatime-cli.zip in this, or different location.

I came back to 6.0.2 for now, thanks in advance for your quick fix!

**Comment by alanhamlett** [comment]
Should be fixed with v6.0.4. However, can you share the error message from your vscode dev console window with me before upgrading the plugin?

**Comment by alanhamlett** [comment]
Should be fixed with v6.0.4. Does the above error message also have a line number from wakatime's extension.js?

**Comment by QuentiumYT** [comment]
Thanks, it is fixed, I didn't had a line number that stands out as it was the whole extension manager that failed to load.
